Packs/CiscoAMP/Integrations/CiscoAMP/CiscoAMP.py

```diff
@@ -0,0 +1,3532 @@
+        args (Dict[str, Any]): Arguments passed down by the CLI to provide in the HTTP request.
+
+    Raises:
+        ValueError: If the input query isn't an IP, URL
+
+    Returns:
+        CommandResults: Information about computers with query activity on them.
+    """
+    query_string = args['query_string']
+    page = arg_to_number(args.get('page', 0))
+    page_size = arg_to_number(args.get('page_size', 0))
+    limit = arg_to_number(args.get('limit', 0))
+
+    filename_regex = r'[\w\-\.]+[\w\-\. ]*'
+
+    # Check if the query is empty or of one of the following formats: SHA256, IPv4, URL or Filename.
+    if is_query_wrong(query_string) \
+            and not bool(re.match(filename_regex, query_string)):
+        raise ValueError('query_string must be: SHA-256/IPv4/URL/Filename')
+
+    pagination = get_pagination_parameters(page, page_size, limit)
+    raw_response_list: List[Dict[str, Any]] = []
+
+    # Run multiple requests according to pagination inputs.
+    for request_number in \
+            range(pagination.offset_multiplier, pagination.number_of_requests + pagination.offset_multiplier):
+        raw_response_list.append(client.computer_activity_list_request(
+            query_string=query_string,
+            limit=pagination.limit,
+            offset=None if pagination.offset is None else pagination.offset * request_number,
+        ))
+
+        if not raw_response_list[-1]['data']:
+            break
+
+    raw_response: Dict[str, Any] = combine_response_results(raw_response_list, pagination.is_automatic)
+
+    context_output = get_context_output(raw_response, ['links'])
+
+    readable_output = get_results_readable_output(raw_response)
+    readable_output += get_readable_output(
+        response=raw_response,
+        header_by_keys=ACTIVITY_HEADERS_BY_KEYS,
+        keys_to_items_option_1=['data'],
+        title=ACTIVITY_TITLE,
+    )
+
+    return CommandResults(
+        outputs_prefix='CiscoAMP.ComputerActivity',
+        outputs_key_field='connector_guid',
+        outputs=context_output,
+        readable_output=readable_output,
+        raw_response=raw_response,
+    )
+
+
+def computers_isolation_feature_availability_get_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+    Get information about available isolation options for a computer.
+
+    Args:
+        client (Client): Cisco AMP client to run desired requests
+        args (Dict[str, Any]): Arguments passed down by the CLI to provide in the HTTP request.
+
+    Returns:
+        CommandResults: Information about a computer's isolation options.
+    """
+    connector_guid = args['connector_guid']
+
+    try:
+        raw_response = client.computer_isolation_feature_availability_get_request(
+            connector_guid=connector_guid,
+        )
+        readable_output = get_isolation_options_readable_output(raw_response)
+
+    except DemistoException as exc:
```

Comment:
Kind of, this command's purpose is to acknowledge the user what isolation commands are viable.
In case isolation isn't viable the API returns an error and with that the response that none option is viable.

Should I change this to raise an exception?

Reply:
I see, it makes sense, I'll suggest a solution directly there